test/extended/fixtures/forcepull-false-cust-bldr.json

```diff
@@ -0,0 +1,32 @@
+{
+    "kind": "BuildConfig",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "forcepull-false-cust-bldr",
+        "creationTimestamp": null,
+        "labels": {
+            "name": "forcepull-false-cust-bldr"
+        }
+    },
+    "spec": {
+        "triggers": [],
+        "source": {
+            "type": "Git",
+            "git": {
+		"uri": "https://github.com/gabemontero/cust-forcepull-false-origin-custom-docker-builder.git"
```

Comment:
@bparees and I talked off line about this one as well ... I will do some consolidation of repos, imagestream, projects, but will keep separate builder images and not worry about having to reset the builder images between tests.